packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -155,34 +158,53 @@ contract OPStackManager is ISemver {
     /// focused on an OPSM version that unblocks interop, we are not proxying OPSM for simplicity.
     /// Later, we will `_disableInitializers` in the constructor and replace any constructor logic
     /// with an `initialize` function, which will be a breaking change to the OPSM interface.
-    constructor(SuperchainConfig _superchainConfig, ProtocolVersions _protocolVersions, Blueprints memory _blueprints) {
-        // TODO uncomment these as we add more validations to this contract, currently this will break a few tests.
-        // assertValidContractAddress(address(_superchainConfig));
-        // assertValidContractAddress(address(_protocolVersions));
-        // assertValidContractAddress(_blueprints.addressManager);
-        // assertValidContractAddress(_blueprints.proxy);
-        // assertValidContractAddress(_blueprints.proxyAdmin);
-        // assertValidContractAddress(_blueprints.l1ChugSplashProxy);
-        // assertValidContractAddress(_blueprints.resolvedDelegateProxy);
+    // constructor(SuperchainConfig _superchainConfig, ProtocolVersions _protocolVersions) {
```

Comment:
Using initializer pattern instead of constructor now that OPSM is proxied. Non-value types can't be immutables so thought I'd remove constructor initialization all together.

Reply:
Resolving. Reverted back to keeping immutables here.